DataManagementSystem/Client/FTSJob.py

```diff
@@ -729,9 +729,15 @@ def toSQL( self ):
 
     :return: str with SQL fragment
     """
-    colVals = [ ( "`%s`" % column, "'%s'" % value if type( value ) in ( str, datetime.datetime ) else str( value ) )
-                for column, value in self.__data__.items()
-                if value and column not in  ( "FTSJobID", "LastUpdate" ) ]
+    colVals = []
```

Comment:
ok.
I did not have time to do my PR, can include in this one the diff i sent you this morning please ?